ur_moveit_config/doc/index.rst

```diff
@@ -23,3 +23,16 @@ interaction using
 
 Now you should be able to use the MoveIt Plugin in rviz2 to plan and execute trajectories with the
 robot as explained `here <https://moveit.picknik.ai/main/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>`_.
+
+.. note::
+   Usually, MoveIt uses trajectory execution monitoring (TEM). If executing a trajectory takes too long,
+   e.g. because the action server died or the robot is blocked, this will lead to an error in
+   trajectory execution. However, with the scaled joint trajectory controller used by this driver,
+   this doesn't make much sense (as long as TEM is not aware of the scaling), as execution time can
```

Comment:
Perhaps "does not work" or "is incompatible with".

It still makes sense, it's just that with the current implementation on MoveIt's side it will result in undesirable behaviour, as it always sees execution time violations.

Reply:
I did not want to place TEM as a bad idea or anything, so "doesn't make sense" isn't the best wording, yes. "does not work" also isn't entirely true, as it does work in terms that is monitors whether the trajectory is executed in the expected time. Howabout "isn't necessarily compatible with"? I mean, it is pretty compatible with it, as long as no scaling is happening.